WooCommerce/Classes/ViewRelated/ReusableViews/TitleAndSubtitleTableViewCell.swift

```diff
@@ -0,0 +1,46 @@
+import UIKit
+
+final class TitleAndSubtitleTableViewCell: UITableViewCell {
+
+    @IBOutlet var subtitleLabel: UILabel!
+    @IBOutlet var titleLabel: UILabel!
+    @IBOutlet var statusContainerView: UIView!
+    @IBOutlet var statusLabel: PaddedLabel!
```

Comment:
nit: could these views stay private and have either a simple view model or function to update the text & other styles for each cell?

Reply:
I've added a separate view model for the cell and made the properties private in 2fa7ddb1466f9b6f5943b935e9df144e76a44b97.